flight_manager.py

```diff
@@ -125,10 +131,7 @@ def run_threads(self, sim: bool) -> None:
         except KeyboardInterrupt:
```

Comment:
In your file, you guys have the SystemExit error along with KeyboardInterrupt, is that not needed here?